src/openstudio_app/StartupView.cpp

```diff
@@ -139,7 +139,7 @@ StartupView::StartupView(QWidget* parent) : QWidget(parent), m_templateListModel
   cancelButton->setStyleSheet("QPushButton { font: bold; }");
 
   auto chooseButton = new QPushButton();
-  chooseButton->setObjectName("StandardBlueButton");
+  chooseButton->setObjectName("StandardGrayButton");
```

Comment:
StartupView is dead code actually... I realized that today. Its not used anywhere.

https://github.com/openstudiocoalition/OpenStudioApplication/blob/f36960c500b4d32de8f9a5a174f0ac563f9ef744/src/openstudio_app/OpenStudioApp.cpp#L926-L936